doc/developer/nxeps/nxep-0003.rst

```diff
@@ -149,27 +149,27 @@ Usage and Impact
 
 Users will build graphs using similar syntax as before with added flexibility.
 
-Create a wheel graph with 9 spokes (10 nodes):
+Create a wheel graph with 9 spokes (10 nodes)::
 
     >>> G = nx.wheel_graph(9)  # same as current code
 
-Construct a path graph using a MultiDiGraph data structure:
+Construct a path graph using a MultiDiGraph data structure::
```

Comment:
Making the following lines runnable in doctests should cause them to fail, since the Graph generator method syntax isn't actually implemented, right? In other words, the lines like `>>> MDG = nx.path_graph.MultiDiGraph([3, 4, 2, 5, 7, 6])` should be kept in blocks that have syntax highlighting (which the indenting accomplishes) but don't run in doctest mode.

Reply:
Yes, these lines do fail the doctest.  But without the double colon, they fail due to namespace discontinuity rather than unimplemented functions.  So the errors are things like G not existing rather than new code features.  What is the best way to handle this kind of document?  Maybe we should remove the >>> completely to avoid doctest and use indenting to preserve code-block and syntax highlighting?

Reply:
You are right ... Adding the double colons doesn't help these files pass any doctests. It unifies the doctests to a single namespace, but that doesn't halp them pass. I think I should leave the NXEP files alone to reduce "blame noise". :}